.prettierignore

```diff
@@ -64,5 +64,6 @@ packages/upgrade/**/*.output.tsx
 # Accessibility Verification Testing
 **/.avt/**
 
-# CODEOWNERS
+# .github files
 CODEOWNERS
+PULL_REQUEST_TEMPLATE.md
```

Comment:
Unrelated to the proposal in this pull request, I noticed that the final newline was deleted here. Do you have [EditorConfig for VS Code](https://marketplace.visualstudio.com/items?itemName=EditorConfig.EditorConfig) installed? The `.editorconfig` file in the codebase includes this rule:

https://github.com/carbon-design-system/carbon/blob/dc698e29ce768b0415fe6ade56db2723a938a155/.editorconfig#L11

However, that rule won’t actually take effect in VS Code unless the extension is installed.

Reply:
Weird, vscode's settings sync didn't pull this through to my new machine for some reason. Thanks for pointing it out.